addons/website_forum/models/forum.py

```diff
@@ -129,6 +131,25 @@ def _get_default_faq(self):
     karma_post = fields.Integer(string='Ask questions without validation', default=100)
     karma_moderate = fields.Integer(string='Moderate posts', default=1000)
 
+    @api.depends('post_ids')
+    def _compute_last_post_id(self):
+        for forum in self:
+            last_post = forum.post_ids.search([('parent_id', '=', False)], order='create_date desc', limit=1)
```

Comment:
please check query :/ and optimize

Reply:
I optimize the code but about the query, I don't see what I could change